x/group/server/msg_server.go

```diff
@@ -457,6 +457,32 @@ func (s serverImpl) CreateProposal(ctx types.Context, req *group.MsgCreatePropos
 		return nil, err
 	}
 
+	// Try to execute proposal immediately
+	if req.TryExec {
+		// Consider proposers as Yes votes
+		for i := range proposers {
```

Comment:
yes, how about `2*loop_step`?

Reply:
@atheeshp @robert-zaremba what about using something similar to what we have in feegrant for now? https://github.com/cosmos/cosmos-sdk/blob/25ecec6c152aa6406da9da24151f412384a3d94b/x/feegrant/filtered_fee.go#L70

Reply:
`gasCostPerIteration` is the `loop_step` in my comment. Usually it should be 10 (as in your code).
Here I suggest to double it, because do more things here.

Reply:
ok got it